lib/new_version.dart

```diff
@@ -77,10 +77,16 @@ class NewVersion {
   /// See http://en.wikipedia.org/wiki/ ISO_3166-1_alpha-2 for a list of ISO Country Codes.
   final String? iOSAppStoreCountry;
 
+  /// Only use for testing purpose. If you ues [forceAppVersion] then `getVersionStatus()`
+  /// always return [storeVersion] equal to [forceAppVersion]. It is can be used
+  /// to test by manually setting app version.
+  final String? forceAppVersion;
```

Comment:
Can we go with:
> /// An optional value that will force the plugin to always return [forceAppVersion] as the value of [storeVersion].
/// This can be useful to test the plugin's behavior before publishng a new version.